modules/ocf_mirrors/manifests/monitoring.pp

```diff
@@ -7,6 +7,7 @@
     $upstream_path = "/${title}",
     $upstream_protocol = 'http',
     $ts_path = undef,
+    $filename = 'health',
```

Comment:
I think this could be removed again, since none of the health checks use this any more.